src/validations/businessCaseSchema.ts

```diff
@@ -0,0 +1,26 @@
+import * as Yup from 'yup';
+
+//
+const phoneNumberRegex = /( *-*[0-9] *?){10,}/;
```

Comment:
This is kind of a funny regex, but it fits the requirements. We are capturing inputs with at least 10 digits. The digits can be separated by spaces or `-`, but there much be 10 digits in order to match.

Reply:
Should we be allowing phone numbers with 11 digits?